### ISSUE TITLE
Use ingress instead of port forward for better stability

### ISSUE THREAD

**Issue by zezhong-zhang** [deepmodeling/dflow#453]
# Summary

Currently user has to explicitly forward the port in the command line. Sometimes the notebook submission fails to find the minio port and cannot submit the jobs, while the `netstat -tulpn | grep :9000 ` shows the port is being use by kubectl. In this case, one has to kill the kubectl process and restart the port forward, then the submission works again. This problem does not have a fixed pattern to reproduce and is causing instability. 

# Use Cases

Maybe better to use [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)?

**Comment by zjgemi** [comment]
After running `minikube addons enable ingress`, you will get this message on the screen
```
After the addon is enabled, please run "minikube tunnel" and your ingress resources would be available at "127.0.0.1"
```
By default, Minikube network is different from the local one. You cannot access the Minikube network from the local one without tunneling. Thus `minikube tunnel` command is required to keep running as well. `port-forward` basically equals `ingress` plus `tunnel`.
For better stability, you may add `port-forward` command as a system service, so that `systemd` will daemon its running.

**Comment by zjgemi** [comment]
https://github.com/kubernetes/minikube/issues/13951
